.gitignore

```diff
@@ -251,3 +251,5 @@ tags
 
 ### MATLAB ###
 *.mex*
+HELICS
```

Comment:
@trevorhardy  did you make this change if so why?

Reply:
I don't remember that change; I may have no need to add .mex to .gitignore and may have done so in error.

Reply:
I was more wondering about the `HELICS` since that and the .DS_Store were added in this PR.

Reply:
Oh, That "HELICS" is probably the Xcode project I set up though I may have added it incorrectly to the .gitignore so that it would be more clearly specific to my installation. Do whatever you need to do for the merge and I'll clean it up on my end as needed.

Reply:
I removed the HELICS